pcsx2/gui/AppInit.cpp

```diff
@@ -167,16 +167,25 @@ void Pcsx2App::AllocateCoreStuffs()
 			{
 				scrollableTextArea->AppendText( L"* microVU0\n\t" + ex->FormatDisplayMessage() + L"\n\n" );
 				recOps.UseMicroVU0	= false;
+#ifndef DISABLE_SVU
 				recOps.EnableVU0	= recOps.EnableVU0 && m_CpuProviders->IsRecAvailable_SuperVU0();
+#else
+				recOps.EnableVU1	= false;
```

Comment:
If SVU is enabled then the value of recOps.EnableVU0 is changed, but disabled recOps.EnableVU1 is changed.

Is that correct?